ydb/public/sdk/cpp/client/ydb_table/table.cpp

```diff
@@ -2384,6 +2384,9 @@ TGlobalIndexSettings TGlobalIndexSettings::FromProto(const TProto& proto) {
     };
 }
 
+// explicit instantiations to avoid defining the template in the .h file
+template TGlobalIndexSettings TGlobalIndexSettings::FromProto(const Ydb::Table::GlobalIndexSettings& proto);
```

Comment:
Move this function to header sounds better to me

Reply:
Maybe even make it non-template, because the only type it currently uses is Ydb::Table::GlobalIndexSettings. Moreover, SerializeTo can only serialize it to the aforementioned protobuf.